src/Shared/IsExternalInit.cs

```diff
@@ -0,0 +1,6 @@
+namespace System.Runtime.CompilerServices
```

Comment:
license :)

And lets add a comment about this being used for init properties for libraries that target frameworks lower than net5.0

Reply:
oops sorry. i'm not used to license so missed it again :)

added it and comment as well